moveit_core/CMakeLists.txt

```diff
@@ -21,8 +21,19 @@ endif()
 
 find_package(Boost REQUIRED system filesystem date_time thread iostreams)
 find_package(Eigen3 REQUIRED)
- # TODO: Move collision detection into separate packages
-find_package(Bullet REQUIRED)
+
+# TODO: Move collision detection into separate packages
+find_package(Bullet CONFIG)
+if(BULLET_FOUND AND BULLET_VERSION_STRING VERSION_GREATER "2.86")
+  set(BULLET_ENABLE "BULLET")
+  set(BULLET_INCLUDE_DIRS "${BULLET_ROOT_DIR}/${BULLET_INCLUDE_DIRS}")
+  set(BULLET_INCLUDE_DIR "${BULLET_ROOT_DIR}/${BULLET_INCLUDE_DIR}")
+  set(BULLET_LIBRARY_DIRS "${BULLET_ROOT_DIR}/${BULLET_LIBRARY_DIRS}")
+  set(BULLET_LIBRARIES "libBulletDynamics.so;libBulletCollision.so;libLinearMath.so;libBulletSoftBody.so")
+  message(STATUS "Compiling with Bullet")
+else()
+  message(STATUS "Version of Bullet too old or not available: disabling Bullet collision detection plugin")
```

Comment:
```suggestion
  message(STATUS "Version of Bullet too old or not available: disabling Bullet collision detection plugin. Try using ROS Melodic or newer.")
```

Reply:
Providing solutions in error messages is master class

Reply:
added.